install

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I'd prefer that skopos stood alone. That it didn't depend upon shell config and assume things about a users home directory setup. I have set a subconscious time go routine in my mind to mull over a concrete recommendation on getting there.

Reply:
had OOB conversation. Good chat. I'll be thinking through other possibles as well as looking at other projects' methodologies for ideas.